### PR TITLE
Update to react-router-redux

### LINKED ISSUE #116: redux-simple-router has been renamed to react-router-redux
See https://github.com/rackt/react-router-redux/issues/80

### PR DESCRIPTION
**Pull request by jbinto**
Fix #116.

See rackt/react-router-redux#80

### CODE REVIEW

**Comment by mxstbr** [comment]
Thanks a lot!

File: package.json [hunk 1]
```diff
@@ -52,6 +51,7 @@
     "postcss-reporter": "^1.3.0",
     "psi": "^2.0.2",
     "react-addons-test-utils": "^0.14.6",
+    "react-router-redux": "^2.1.0",
```
Comment (jcano) [inline package.json]: Is there a reason for adding react-router-redux on the `devDependencies` list instead of `dependencies`?

**Comment by jcano** [comment]
On the package.json file, is there a reason for adding react-router-redux on the `devDependencies` list instead of `dependencies`?

**Comment by mxstbr** [comment]
Absolutely none @jcano, would you mind fixing that and submitting a PR?

**Pull request merged**